scripts/generate-configs

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash ../shell.nix
+
+# Usage: generate-configs OUTPUT
+#
+# Generates configuration for all services found in BOTOCORE_PATH that
+# are missing corresponding configurations in ./config/services.
+#
+# The example configurations will be written to individual files under
+# OUTPUT/configs and OUTPUT/annexes.
+
+set -euo pipefail
+
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+botocore="${BOTOCORE_PATH?}"
+services="config/services"
+output="${1?An OUTPUT argument is required}"
+configs="$output/configs"
+annexes="$output/annexes"
+
+available=()
+configured=()
+unconfigured=0
+
+# Output a message to stderr.
+say() {
+  echo >&2 "$@"
+}
+
+# Write an example configuration and annex to name.json.
+write_config() {
+  local -r name="$1"
+  local -r config="$configs/$name.json"
+  local -r annex="$annexes/$name.json"
+
+  say "Missing $name"
+
+  jq -n --arg name "$name" '{ libraryName: $name }' >"$config"
+  jq -n --arg name "$name" '{}' >"$annex"
```

Comment:
What's this for? Won't it always return `{}`?

Reply:
Yeah, it will create an empty annex file per:

> The reason for also generating an empty annex file ..

It's just more convenient to pre-create the file since the majority of new service setups needs `serviceAbbreviation` added there.

Reply:
I meant the use of `jq` over `echo {} > "$annex"` (using `jq` for symmetry, perhaps?)

Reply:
Ah, yeah, no other reason.